clouddriver-sql/src/main/kotlin/com/netflix/spinnaker/clouddriver/sql/SqlTaskRepository.kt

```diff
@@ -295,7 +295,7 @@ class SqlTaskRepository(
 
       val select = if (thisInstance) {
         baseQuery
-          .innerJoin(tasksTable.`as`("t")).on(field("a.task_id").eq("t.id"))
+          .innerJoin(tasksTable.`as`("t")).on(sql("a.task_id = t.id"))
```

Comment:
`.on(field("a.task_id").eq(field("t.id"))`?

Reply:
Yes, that was the first thing i wrote. But it has:
`Overload resolution ambiguity: 
public abstract fun eq(p0: Any!): Condition! defined in org.jooq.Field
public abstract fun eq(p0: Field<Any!>!): Condition! defined in org.jooq.Field`
And I don't have a good idea how to fix it. 

Also we already have `on(sql("a.task_id = b.task_id and a.created_at = b.created"))` in this query and maybe for the same reason.